packages/ui/src/bounty/components/BountiesTiles.tsx

```diff
@@ -31,7 +31,7 @@ export const BountiesTiles = ({ onlyContributed = false }: Props) => {
         },
         {
           title: t('tiles.earned.title'),
-          content: <TokenValue value={statistics.amountEarned} size="l" />,
+          content: <TokenValue value={new BN(statistics.amountEarned ?? 0)} size="l" />,
```

Comment:
@stepien4soft actually please still move the number to BN casting to @/bounty/components/BountiesTiles.tsx.
Once https://github.com/Joystream/pioneer/issues/3285 will be fixed it will become string to BN so the code won't change.